company_linkage/sql/linked_all_patents.sql

```diff
@@ -0,0 +1,35 @@
+-- Pulling every patent family id linked to every ror id of any assignee for that patent, and all the assignee names
+-- We also attempt to add in "fake" families for the patents that are missing patent families
+with patents_orig as (
+SELECT
+  -- Pulling in the current assignee ror ids
+  patent_id,
+  family_id,
+  assignee,
+  ror_id
+FROM
+  unified_patents.assignees_normalized),
+  patent_years as (
+  SELECT
+      patent_id,
+      EXTRACT(year FROM first_priority_date) as priority_year
+    FROM
+      unified_patents.dates
+  )
+  SELECT
+    DISTINCT
```

Comment:
Do we still need to do this given https://github.com/georgetown-cset/unified-patents/pull/97/files ? (I don't think it necessarily hurts anything but just wanted to double check this is no longer needed)

Reply:
I don't think it hurts, and there are still probably some patents with no family ids, so I'm leaving it in.